packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -30,6 +30,9 @@ const BlockInspector = ( {
 	selectedBlockName,
 	showNoBlockSelectedMessage = true,
 } ) => {
+	const slot = useSlot( InspectorAdvancedControls.slotName );
```

Comment:
Yes, let's keep `__experimentalUseSlot` experimental until we come up with a better way to handle it. It seems fine for the time being.